src/__tests__/SkeletonContent.test.tsx

```diff
@@ -52,35 +52,37 @@ describe('SkeletonComponent test suite', () => {
     expect(bones[0].props.style).toEqual({
       alignItems: 'center',
       flex: 1,
-      justifyContent: 'center'
+      justifyContent: 'center',
```

Comment:
If you don't like trailing commas I can remove this, this is just the new default so I left it in

Reply:
No strong opinion, we can keep it!